commit b1200a60be7bdea20ffebbd8bb15386041727fa6
Author: Olly Betts <olly@survex.com>
Date:   Wed May 18 12:06:36 2022 +1200

    Fix cavern.tst to work with SOURCE_DATE_EPOCH set

diff --git a/tests/cavern.tst b/tests/cavern.tst
index cec23be2..d2b189af 100755
--- a/tests/cavern.tst
+++ b/tests/cavern.tst
@@ -88,6 +88,11 @@ export SURVEXLANG
 LSAN_OPTIONS=leak_check_at_exit=0
 export LSAN_OPTIONS
 
+# Allow datestamps in 3d files (we normalise the expected output for GPX
+# etc) to allow for the datestamp not being fixed, but under SOURCE_DATE_EPOCH
+# the datestamp is omitted entirely which would break those testcases.
+unset SOURCE_DATE_EPOCH
+
 vg_error=123
 vg_log=$testdir/vg.log
 if [ -n "$VALGRIND" ] ; then
